hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieMetadataTableValidator.java

```diff
@@ -1846,7 +1846,7 @@ public List<HoodieColumnRangeMetadata<Comparable>> getSortedColumnStatsList(Stri
             String filePath = new StoragePath(storagePartitionPath, filename).toString();
             try {
               return getLogFileColumnRangeMetadata(filePath, partitionPath, metaClient, allColumnNameList, Option.of(readerSchema),
-                  metadataConfig.getMaxReaderBufferSize())
+                  metadataConfig.getMaxReaderBufferSize(), metadataConfig.isOptimizedLogBlocksScanEnabled())
```

Comment:
This should be write config

Reply:
Will fix this, I think the `props` field of this class may contain this config key, so will check that otherwise fallback to default value.